README.md

```diff
@@ -12,7 +12,8 @@ Node.js module.
   - [This is an alpha 1.x version](#alpha)
   - [Not accepting 0.x fixes anymore](#0.x-fixes)
 - [About Globalize](#about)
-  - [Why globalization?](#why)
+  - [Why globalization?](#why_globalization)
```

Comment:
What do you think of https://github.com/jquery/globalize/pull/392/files#diff-04c6e90faac2675aa89e2176d2eec7d8R80